source/extensions/filters/common/rbac/engine.h

```diff
@@ -27,7 +27,8 @@ class RoleBasedAccessControlEngine {
    * @param metadata   the metadata with additional information about the action/principal.
    */
   virtual bool allowed(const Network::Connection& connection, const Envoy::Http::HeaderMap& headers,
-                       const envoy::api::v2::core::Metadata& metadata) const PURE;
+                       const envoy::api::v2::core::Metadata& metadata,
+                       std::string& effectivePolicyID) const PURE;
```

Comment:
`snake_case` for all function parameters and local variables (throughout)

Reply:
updated.